File: game-client-new/src/main/java/net/dodian/client/GameClient.java
package net.dodian.client;

import com.jagex.runescape.Client;
import com.jagex.runescape.sign.Signlink;
import com.jagex.runescape2.Game;

import java.net.InetAddress;
import java.net.UnknownHostException;

public class GameClient {

    public static void main(String[] args) {
        Game.setHighmem();
        Game.portOffset = 0;
        Game.nodeID = 0;
        Game.members = false;
        Game.crc32Check = false;
        Game.server = "127.0.0.1";

        com.jagex.runescape2.Signlink.storeid = 1;
        com.jagex.runescape2.Signlink.startpriv();

        Game game = new Game();
        game.jaggrabEnabled = false;
        game.init(765, 503);
    }

    public static void main1(String[] args) throws UnknownHostException {
        Client.localWorldId = 0;
        Client.portOffset = 0;
        Client.setHighMem();
        Client.membersWorld = false;

        Signlink.storeid = 1;
        Signlink.startpriv(InetAddress.getLocalHost());

        Client client = new Client();
        client.createClientFrame(765, 503);
    }
}

File: dodian-frontend/dodian-client/src/main/kotlin/net/dodian/uber/client/GameClient.kt
package net.dodian.uber.client

import com.jagex.runescape.Game
import com.jagex.runescape.Signlink
import net.dodian.library.extensions.argument
import java.math.BigInteger

fun main(args: Array<String>) {
    val game = Game()

    Game.RSA_MODULUS = BigInteger("144536637563440057160808461977324382186710417775358184126989832762756698107899344110117559434824610238034809802491751871048292985661698319076446382120631557440502945377688897688601296108720422774470526242807665051371841237960974220302678715599412577220453538331305436281828154659005659234330053472006964242623")
    Game.RSA_EXPONENT = BigInteger("65537")

    Game.server = args.argument("host") ?: "127.0.0.1"
    Game.gamePort = args.argument("gamePort")?.toIntOrNull() ?: 43594
    Game.filePort = args.argument("filePort")?.toIntOrNull() ?: 43595
    Game.httpPort = args.argument("httpPort")?.toIntOrNull() ?: 8080

    Game.nodeID = args.argument("node")?.toIntOrNull() ?: 10
    Game.portOffset = args.argument("portOffset")?.toIntOrNull() ?: 0

    Game.lowmem = args.argument("lowMemory")?.toBooleanStrictOrNull() ?: false
    Game.members = args.argument("members")?.toBooleanStrictOrNull() ?: true

    Signlink.storeid = args.argument("storeId")?.toIntOrNull() ?: -1
    Signlink.startpriv()

    game.init(765, 503)
}